CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ Contributor License Agreement
 - You grant Forge a license to use your code contributed to the primary codebase (everything **not** under patches) in Forge, under the LGPLv2.1 license.
 - You assign copyright ownership of your contributions to the patches codebase (everything under patches) to Forge, where it will be licensed under the LGPLv2.1 license.
 
-Additionally, while 1.9.4 is being maintained, you will be asked to acknowledge these two additional clauses.
+Additionally, while 1.12.2 is being maintained, you will be asked to acknowledge these two additional clauses.
```

Comment:
I don't think this is an typo

Reply:
It isn't a typo, I just changed it to 1.12.2 as it is the Latest Maintained Version.

Reply:
No, they mean that this changes the meaning of the document. The additional two clauses are for the old license which does not apply after 1.10

Reply:
Ah, Ok. In that case, will fix this shortly (MCP Tests were also working so the mappings will be on 20171005 in the next commit

Reply:
Fixed!